fern/docs.yml

```diff
@@ -1704,6 +1704,50 @@ navigation:
             api-name: clankermon
         slug: clankermon
 
+      - section: Stable
+        contents:
+          - api: Stable API Endpoints
+            api-name: stable
+        slug: stable
+
+      - section: Stable
+        contents:
+          - api: Stable API Endpoints
+            api-name: stable
+        slug: stable
+      - section: Tron
+        contents:
+          - api: Tron API Endpoints
+            api-name: tron
+        slug: tron
+
+      - section: Stable
+        contents:
+          - api: Stable API Endpoints
+            api-name: stable
+        slug: stable
+      - section: Tron
+        contents:
+          - api: Tron API Endpoints
+            api-name: tron
+        slug: tron
+
+      - section: MegaETH
+        contents:
+          - api: MegaETH API Endpoints
+            api-name: megaeth
+        slug: megaeth
+      - section: Stable
+        contents:
+          - api: Stable API Endpoints
+            api-name: stable
+        slug: stable
+      - section: Tron
+        contents:
+          - api: Tron API Endpoints
+            api-name: tron
+        slug: tron
+
```

Comment:
Hmm whoops looks like if we let it sit for a few days it'll keep appending the new chains. That's a tricky solve actually. Let's just fix manually for now